src/components/OverviewMap.js

```diff
@@ -24,31 +22,34 @@ class OverviewMap extends React.Component {
       </Popup>) : null);
       if (geojson) {
         contents.push(<GeoJson key={id} data={geojson}>{content}</GeoJson>);
-      } else if (isNumber(latitude) && isNumber(longitude)) {
-        contents.push(<Marker key={id} position={[latitude, longitude]}>{content}</Marker>);
       }
     });
-    if (position === null) {
-      position = [60.192059, 24.945831];  // Default to Helsinki's center
-    }
-    return {position, contents};
+    return contents;
   }
 
   render() {
     if (typeof window === "undefined") return null;
     const {style, hearings} = this.props;
-    const {Map, TileLayer} = require('react-leaflet');  // Late import to be isomorphic compatible
-    const {position, contents} = this.getHearingMapContent(hearings);
+    const {Map, TileLayer, FeatureGroup} = require('react-leaflet');  // Late import to be isomorphic compatible
+    const contents = this.getHearingMapContent(hearings);
     if (!contents.length && this.props.hideIfEmpty) {
       return null;
     }
+    const position = [60.192059, 24.945831];  // Default to Helsinki's center
     return (
       <Map center={position} zoom={13} style={style}>
         <TileLayer
           url="http://{s}.tile.osm.org/{z}/{x}/{y}.png"
           attribution='&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
         />
-        {contents}
+        <FeatureGroup
+          ref={(input) => {
```

Comment:
Okay, I didn't know you can pass callbacks as refs these days :)

(ref https://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute)